### PR TITLE
Add Pod Disruption Budget to Glossary

### PR DESCRIPTION
**Pull request by McCoyAle**
#5993

### CODE REVIEW

File: content/en/docs/reference/glossary/pod-disruption-budget.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+full-link: /docs/concepts/workloads/pods/disruptions/
+aka:
+ - PDB
+related:
+ - pod
+ - container
+tags:
+ - operation
+short-description: >
+ An object that limits the number of Pods of a replicated application, that are down simultaneously from voluntary disruptions.
```
Comment (sftim) [inline content/en/docs/reference/glossary/pod-disruption-budget.md]:
```suggestion
 An object that limits the number of {{< glossary_tooltip text="Pods" term_id="pod" >}} of a replicated application, that are down simultaneously from voluntary disruptions.
```

I think this kind of cross-reference is helpful; however it's also broken right now (see #12608).

Comment (McCoyAle) [reply content/en/docs/reference/glossary/pod-disruption-budget.md]: @sftim I agree. Thank you!

**Comment by zparnold** [comment]
@McCoyAle I made an adjustment based on the style in other glossary entries. Let me know what you think.

**Pull request merged**